src/lm_buddy/cli/schema.py

```diff
@@ -1,5 +1,11 @@
+import json
+
 import click
 
+from lm_buddy.jobs.finetuning import FinetuningJobConfig
```

Comment:
So the folder layout I'm thinking of is as follows:

```
lm_buddy.jobs
   configs/
   _entrypoints
   __init__.py
```

Then, the individual entrypoint functions could be better shielded from the public API since we only expose the generic `run_job` method.

Reply:
I think ill do this re-organization in a follow up PR to avoid the clutter in this one.

Reply:
let's discuss in that second PR, but it would be helpful to see the new workflow too

Reply:
Ill end up adding a new example notebook showing how to execute the library directly, outside of Ray and the CLI.